examples/asr/asr_adapters/train_asr_adapter.py

```diff
@@ -168,9 +181,23 @@ def main(cfg):
     # Then, Unfreeze just the adapter weights that were enabled above (no part of encoder/decoder/joint/etc)
     model.unfreeze_enabled_adapters()
 
+    # Update model config prior to training
+    model.cfg = model.cfg
```

Comment:
I don't understand what does this line do?

Reply:
Its a copy of the internal config to the PTL cache, so that WandB will show the updated config in logs. Without this, configs will not update. Its somewhat a limitation of PTLs internal wandb log handling step.